allure-testng/src/main/java/io/qameta/allure/testng/config/AllureTestNgConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.qameta.allure.testng.config;
+
+import io.qameta.allure.util.PropertiesUtils;
+
+import java.util.Properties;
+
+public class AllureTestNgConfig {
+
+    public static final String ALLURE_TESTNG_CONFIG = "allure.testng.config.";
```

Comment:
omit the `config.` suffix as unnecessary

Reply:
Cool, I've changed it to "allure.testng.hide.disabled.tests"